cloud-aws/src/main/resources/templates/aws-cf-dbstack.ftl

```diff
@@ -89,7 +89,7 @@
     "VPCSecurityGroupsParameter": {
         "Type": "List<AWS::EC2::SecurityGroup::Id>",
         "Description": "VPC security groups"
-    },
+    }
```

Comment:
Is this a bug being fixed? What does it cause if we have the `,` there?

Reply:
Yes, it is a malformed JSON. I noticed this when adding a unit test case. I wonder if anyone has used that code path before, otherwise CF would have rejected the request (unless their JSON parser is lenient enough to ignore such extra commas).

Reply:
In particular, each test case of `CloudFormationTemplateBuilderDBTest` starts with the assertion `assertThat(JsonUtil.isValid(result)).overridingErrorMessage("Invalid JSON: " + result).isTrue();`. This failed due to the above extra comma.